MyHabits/InfoViewController/InfoViewController.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  InfoViewController.swift
+//  MyHabits
+//
+//  Created by Anton Kislov on 05.06.2023.
+//
+
+import UIKit
+
+class InfoViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupViews()
+        view.backgroundColor = .systemBackground
+        title = "Информация"
+    }
+    
+    // MARK: Creating scrollView
+    lazy var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.keyboardDismissMode = .interactive
+        return scrollView
+    }()
+    
+    lazy var containerView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    func createLable(_ text: String) -> UILabel {
+        let label = UILabel()
+        label.text = text
+        label.textColor = .black
+        label.font = const.bodyFont
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        return label
+    }
+    
+    lazy var headerLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Привычка за 21 день"
+        label.font = const.title3Font
+        label.textColor = .black
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        return label
+    }()
+    
+    lazy var label0: UILabel = {
```

Comment:
You can use 1 label for all the text.
Remember how you can set the new line in the string.

Reply:
you must see, some problem with scroll content. Scroll dont work